crates/gateway-types/src/reply.rs

```diff
@@ -2399,7 +2399,7 @@ mod tests {
             serde_json::from_str::<MaybePendingBlock>(integration::block::NUMBER_319709).unwrap();
             serde_json::from_str::<MaybePendingBlock>(v0_13_0::block::PENDING).unwrap();
             // This is from integration starknet_version 0.13.0 and contains data gas prices.
-            serde_json::from_str::<crate::reply::Block>(integration::block::NUMBER_329543).unwrap();
+            serde_json::from_str::<MaybePendingBlock>(integration::block::NUMBER_329543).unwrap();
```

Comment:
Uh, this hunk seems to be unrelated.

Reply:
I must have done this when rebasing, good catch!